client/src/components/new-request/search.vue

```diff
@@ -1,13 +1,16 @@
 <template>
   <v-container fluid id="new-request-container" class="copy-normal pa-10">
-    <v-row cols="12">
+    <v-row no-gutters>
       <v-col cols="6" class="pt-0 font-weight-bold h6"><span>I need a name to:</span></v-col>
       <v-col cols="6" class="pt-0">
         <span id="nr-required-activator"
               class="link-std d-flex justify-end"
               @click="activateNRRModal()">Check to see if you need to file a Name Request</span>
       </v-col>
-      <v-col cols="5" class="mt-n1">
+    </v-row>
+
+    <v-row class="mt-4" no-gutters>
```

Comment:
No Gutters gets rid of margins and padding for the row and cols.

So below (line 42), I added some inter-col padding back in.